tests/baselines/reference/enumAssignmentCompat3.errors.txt

```diff
@@ -90,11 +96,14 @@ enumAssignmentCompat3.ts(86,1): error TS2322: Type 'Merged.E' is not assignable
     abc = secondCd; // missing 'd'
     ~~~
 !!! error TS2322: Type 'Cd.E' is not assignable to type 'First.E'.
-!!! error TS2322:   Property 'd' is missing in type 'First.E'.
+!!! error TS2322:   The values of 'E.c' differ in their declarations, where '2' was expected but '0' was given.

```

Comment:
This seems like a regression - I don't know why the new error message takes precedence here though.

Reply:
Ah, because it just goes property-by-property and whichever goes wrong first wins. Makes sense.